build.gradle

```diff
@@ -31,7 +31,7 @@ allprojects {
     apply plugin: 'jacoco'
 
     group = 'com.uber.jaeger'
-    version = '0.20.0-SNAPSHOT'
+    version = System.env.TRAVIS_TAG ? System.env.TRAVIS_TAG : '0.20.0-SNAPSHOT'
```

Comment:
At the current state, this is not a problem. Either this will fail during the upload phase, or we could just not close/release the staging repo at Nexus.